tests/api/activitypub/actor.test.js

```diff
@@ -0,0 +1,56 @@
+require('should')
+const { createUsername, createUserOnFediverse } = require('../fixtures/users')
+const { startServerWithEmitterAndReceiver, startServerWithEmitterUser, createReceiver, makeUrl } = require('../utils/activitypub')
+const { shouldNotBeCalled, rethrowShouldNotBeCalledErrors, signedReq } = require('../utils/utils')
+
+describe('activitypub:actor', () => {
+  it('should reject unknown actor', async () => {
+    try {
+      const emitterUser = await createUserOnFediverse()
+      const { origin, query } = await startServerWithEmitterUser({ emitterUser })
+      const keyUrl = makeUrl({ origin, params: query })
+      const imaginaryReceiverUsername = createUsername()
+      const receiverUrl = makeUrl({ params: { action: 'actor', name: imaginaryReceiverUsername } })
+      await signedReq({ url: receiverUrl, keyUrl, privateKey: emitterUser.privateKey })
```

Comment:
The internals of `signedReq` (which user from what server are doing the signed request) aren't so relevant to those tests, it would be nice if `signedReq` could hide those (like for instance`adminReq` is doing) by keeping a fediverse server and user at hand, and default to those when not told otherwise
```suggestion
      const imaginaryReceiverUsername = createUsername()
      const receiverUrl = makeUrl({ params: { action: 'actor', name: imaginaryReceiverUsername } })
      await signedReq({ url: receiverUrl })
```